### DISCUSSION TITLE
Question: relationship with gRPC

### DISCUSSION THREAD

**Discussion by cetanu** [eigenein/protobuf#93] [Ideas]
Just found your project, looks really cool.

I am wondering, is there an intention to do so, or does this library already allow one to create a grpc server using the `@message` decorated dataclasses?

Is there a person that is doing this currently that could show an example, if that is the case?

**Comment by eigenein** [comment]
Hi Vasili,

There's no any (g)RPC at the moment, only serialization is implemented.

Even though I looked at gRPC some time ago, I never really used it. So speaking of intention, yes, but I'd need to allocate quite some time for that and this is unlikely in short term. I'll keep the issue to keep track of this
